docs/DeveloperGuide.md

```diff
@@ -221,7 +222,65 @@ for subsequent prompts.
 ---
 
 ### Parsing User Input and Getting the Right Command
-**_TODO_**: Explain how the app parses user input and determines which `Command` subclass object to instantiate.
+
+![Obtain and Parse User Input](uml/sequenceDiagrams/miscellaneous/images/obtainAndParseUserInput.png)
+
+**(Steps 1 and 2)** When a user enters something into the terminal (when prompted), `UI#getUserInput()` will take in the user's input
+as a `String`, use `String#trim()` to remove leading and trailing whitespaces in the input.
```

Comment:
Perhaps "...._as a `String` and call the `String#trim()` method to remove......_"

Reply:
Thanks for spotting the errata. It has been fixed in Commit d9b1029